Code/GraphMol/Wrap/rough_test.py

```diff
@@ -4892,6 +4892,21 @@ def testSVGParsing(self):
       mol = Chem.MolFromRDKitSVG("bad svg")
 
 
+  def testSetQuery(self):
+    pat = Chem.MolFromSmarts("[C]")
+    self.assertFalse(Chem.MolFromSmiles("c1ccccc1").HasSubstructMatch(pat))
+
+    q = rdqueries.AtomNumEqualsQueryAtom(6)
+    for atom in pat.GetAtoms():
+      atom.SetQuery(q)
+
+    self.assertTrue(Chem.MolFromSmiles("c1ccccc1").HasSubstructMatch(pat))
+
+  def testGitHub1985(self):
+    # simple check
+    Chem.MolToSmarts(Chem.MolFromSmarts("[C@]"))
```

Comment:
What is this test intended to check?
It's not currently really doing anything

Reply:
This caused an exception in previous implementations, but I'll go ahead remove it.  I didn't know if I had accidentally deleted it at some point.

Reply:
It's fine to stay in there, but at least a comment about it having previously thrown an exception would be cool